pipeline/metadata/flatten_hyperquack.py

```diff
@@ -135,6 +153,11 @@ def _process_hyperquack_v1(self, filename: str, scan: Any,
       if 'Error' in result:
         row.error = result['Error']
 
+      row.outcome = classify_hyperquack_outcome(
```

Comment:
I worry that we are losing the ability to easily regenerate the outcome.
Perhaps you should make this a separate step, a PTransform that is adds the outcome row. This way we could reuse the same PTranform to update the data, with the existing data as input.

Regardless, I don't think there's significant cost in making this a separate transform, but you will get better metrics and the semantics is clearer. This should run on the output of process_hyperquack. Nesting here makes the processing step harder to understand.

Reply:
Broke off a refactoring I need for this into https://github.com/censoredplanet/censoredplanet-analysis/pull/153

Reply:
Done, moved outcome classification to the end of `process_hyperquack_lines`